lib/getstats.js

```diff
@@ -1,13 +1,18 @@
-/* global RTCRtpTransceiver */
 'use strict';
 
 var flatMap = require('./util').flatMap;
 var guessBrowser = require('./util').guessBrowser;
+var getSdpFormat = require('./util/sdp').getSdpFormat;
 
 var guess = guessBrowser();
 var isChrome = guess === 'chrome';
 var isFirefox = guess === 'firefox';
 var isSafari = guess === 'safari';
+var sdpFormat = getSdpFormat();
+
+var chromeMajorVersion = isChrome
+  ? parseInt(navigator.userAgent.match(/Chrome\/([0-9]+)/)[1], 10)
```

Comment:
The `navigator.userAgent.match(/Chrome\/([0-9]+)/)` gives and error:
```
TypeError: Cannot read property '1' of null
```
specifically when Chrome 72 on MacOS 10.13.6 is in mobile browser mode via Developer Tools. The `guessBrowser()` call still returns `'chrome'` but the `navigator.userAgent` returns this:
```
"Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E304 Safari/602.1"
```
this leads to `null` being returned from the `.match()` call and in turn the `parseInt` function returns `NaN`.

This occured when I ran `npm install twilio-video` in my Vue project and then attempted to import the library as such: `import { connect } from 'twilio-video'`.

Reply:
@manjeshbhargav

Reply:
Hi @srobertson421 ,

Thank you for bringing this to our notice. The user agent you have specified should match to Safari. The reason why you are seeing this behavior is that we determine that the browser is Chrome if `webkitRTCPeerConnection` is defined. When Chrome is emulating Safari, because of this variable, we mistakenly detect the browser as Safari. If you load the SDK on a real iOS Safari browser, it will properly be detected.

The next version will determine the browser based on the user agent instead of `webkitRTCPeerConnection` and should correctly match to Safari when you're running Chrome to emulate iOS Safari.

Thanks,

Manjesh Malavalli
JSDK Team